sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveUtils.scala

```diff
@@ -80,10 +80,8 @@ class HiveContext private[hive](
   protected[sql] override def sharedState: HiveSharedState = {
     sparkSession.sharedState.asInstanceOf[HiveSharedState]
   }
-
```

Comment:
Hm, how would it resolve the conflict if it shows as a diff vs master now? this means the line was present in master.